content/articles/threads-and-isolates-in-flutter/index.md

```diff
@@ -0,0 +1,210 @@
+### Threads and Isolates In Flutter Apps 
+### Introduction
+Threads and isolates are two of the essential abstractions you will need to know when developing apps in Flutter. They allow developers to utilize multicore processors, improve user experience by reducing UI stuttering, and reduce battery consumption on mobile devices. 
+### Table of contents
+- Introduction
+- Table of contents
+- Prerequisites
+- Topics to handles
+- Isolates in Flutter
+- Application Development 
+- Step 1 - Creating States
+- Step 2 -Sending messages through Isolate
+- Step 3 - Passing parameters between threads
+- Step 4 - Serialization
+- Step 5 - Provide Function
+- Calls
+
+### Prerequisites
+- Visual Studio or any code editor installed.
+- Have an Understanding of Flutter Widgets.
+- Familiar with dart programming languages.
+- Know coding flutter apps. 
+### Topics to handle
+This article describes how to use the Dart Thread and Isolate classes in your Flutter application. The goal is to give you a deeper understanding of Thread and Isolate and when appropriate to use them.
+
+### Isolates in Flutter: 
+You can use Isolate together with threads to improve the performance of your application and reduce battery consumption on mobile devices. Isolates are similar to other programming languages' threading concepts, but they have a few key differences.
+
+You can use the Isolate API to spawn new isolates; you can pass messages between an isolate and its parent thread(s). However, you are not directly accessing shared mutable states from within an isolate.
+### Application Development 
+To demonstrate their usage, we will build a simple counter-demo: An application that starts with an initial value at 0 and allows the user to increment or decrement that value.
+
+The application has a button that will start a new counter. When the corresponding Isolate is created, we store its
+instance in a variable and pass it through various message handlers to update the current counter value. The instance of the isolated thread is stored in a class member variable called _isolate.
+
+### Step 1 - Creating States
+- In the following code snippets, a comment starting with // [START isolate] means that an isolated thread is started. A comment beginning with // [END isolate] means the corresponding isolation is terminated.
```

Comment:
You can explain this in a better way. Talk about in this step we will be creating the states. Talk about the methods being used and then complete with the statement you have written.